Sprint-1/4-stretch-explore/objects.md

```diff
@@ -5,12 +5,21 @@ In this activity, we'll explore some additional concepts that you'll encounter i
 Open the Chrome devtools Console, type in `console.log` and then hit enter
 
 What output do you get?
+    undefined
```

Comment:
I think it will actually show function information - you may have typed `console.log()` which will _call_ the function and return `undefined` :)